### DISCUSSION TITLE
Epub file open with error font.

### DISCUSSION THREAD

**Discussion by thanhnpm** [sumatrapdfreader/sumatrapdf#3628] [Q&A]
Hi all,
When I open Vietnamese epub, the font is error. but the bookmark font is ok.
I research and found the way to fix it is to change font with EbookUI like in here. [forum](https://forum.sumatrapdfreader.org/t/having-e-book-character-alternative-font-letter-problems/799)
But seem it not work anymore with the latest version?

![Error](https://github.com/sumatrapdfreader/sumatrapdf/assets/67894924/d59ec7ed-d179-4eb5-9d2f-b122ab2391a4)

Thanks

**Comment by GitHubRulesOK** [comment]
the last version that used ONE custom font at a time was 3.2 (it has the debug menu where you can flip between eBook User Interface) 
You would ideally need the same font as defined in the ebook to be added from system fonts, as per your link

**Comment by kjk** [comment]
Can't investigate without a file that shows the problem.

Also should be reported as bug, not a discussion.
